src/style.css

```diff
@@ -60,3 +60,11 @@ input {
   gap: 30px;
   align-items: center;
 }
+
+.left i {
+  cursor: pointer;
+}
+
+.line-through {
+  text-decoration: line-through;
+}
```

Comment:
- [OPTIONAL]Even though styling is not a major prerequisite, it's also beneficial to at least maintain a good view when designing because you don't know when your profile may be visited by prospective recruiters and you will want to posit yourself as a detail-oriented guy, one with eyes for design, and the mind that is not satisfy with average :+1: 

See the appearance :point_down: 
![image](https://user-images.githubusercontent.com/99715640/222813886-484685a6-796e-4140-865c-ba63fba1ffca.png)